utils/sysinfo/sysinfo.go

```diff
@@ -133,6 +146,200 @@ func GetNetworkDevices(sysfs sysfs.SysFs) ([]info.NetInfo, error) {
 	return netDevices, nil
 }
 
+// GetHugePagesInfo returns information about pre-allocated huge pages
+// hugepagesDirectory should be top directory of hugepages
+// Such as: /sys/kernel/mm/hugepages/
+func GetHugePagesInfo(sysFs sysfs.SysFs, hugepagesDirectory string) ([]info.HugePagesInfo, error) {
```

Comment:
Can I safely assume that function below are just copied from `machine/machine.go`.

Reply:
yes, it is copied

Reply:
:+1: